packages/aztec.js/src/proof/joinSplitFluid/index.js

```diff
@@ -70,13 +62,13 @@ class JoinSplitFluidProof extends JoinSplitProof {
                 inputNotes: [
                     {
                         ...this.outputNotes[0],
-                        forceMetadata: true,
+                        switchEncoding: false,
                     },
                 ],
                 outputNotes: [
                     {
                         ...this.inputNotes[0],
-                        forceNoMetadata: true,
+                        switchEncoding: false,
```

Comment:
I believe if you implement the switch proposed in the outputCoder module, you can get rid of these magic flags.

Reply:
Yep, removed them